Documentation/upgrading.md

```diff
@@ -1,173 +1,149 @@
+node3.example.com                  Ready     21d       v1.6.6+coreos.1
+node4.example.com                  Ready     21d       v1.6.6+coreos.1
+```
 
-Edit the scheduler deployment to rolling update the scheduler. Change the container image name for the hyperkube.
+## Control Plane
 
-    kubectl edit deployments kube-scheduler -n=kube-system
+Show the control plane daemonsets and deployments which will need to be updated.
 
-Wait for the schduler to be deployed.
+```sh
+$ kubectl get daemonsets -n=kube-system
+NAME                             DESIRED   CURRENT   READY     UP-TO-DATE   AVAILABLE   NODE-SELECTOR                     AGE
+kube-apiserver                   1         1         1         1            1           node-role.kubernetes.io/master=   21d
+kube-flannel                     4         4         4         4            4           <none>                            21d
+kube-proxy                       4         4         4         4            4           <none>                            21d
+pod-checkpointer                 1         1         1         1            1           node-role.kubernetes.io/master=   21d
 
-### kube-controller-manager
+$ kubectl get deployments -n=kube-system
+kube-controller-manager           2         2         2            2           21d
+kube-dns                          1         1         1            1           21d
+kube-scheduler                    2         2         2            2           21d
+```
 
-Edit the controller-manager deployment to rolling update the controller manager. Change the container image name for the hyperkube.
+### kube-apiserver
 
-    kubectl edit deployments kube-controller-manager -n=kube-system
+Edit the `kube-apiserver` daemonset to rolling update the apiserver.
 
-Wait for the controller manager to be deployed.
+```sh
+$ kubectl edit daemonset kube-apiserver -n=kube-system
+```
 
-    $ kubectl get pods -n=kube-system
-    NAME                                       READY     STATUS    RESTARTS   AGE
-    kube-apiserver-vyg3t                       2/2       Running   0          18m
-    kube-controller-manager-1709527928-zj8c4   1/1       Running   0          4m
-    kube-dns-v20-3531996453-0tlv9              3/3       Running   0          28m
-    kube-proxy-8jthl                           1/1       Running   0          28m
-    kube-proxy-bnvgy                           1/1       Running   0          28m
-    kube-proxy-gkyx8                           1/1       Running   0          28m
-    kube-scheduler-2255275287-hti6w            1/1       Running   0          6m
-    kubelet-exe5k                              1/1       Running   0          28m
-    kubelet-p3g98                              1/1       Running   0          28m
-    kubelet-quhhg                              1/1       Running   0          28m
-    pod-checkpointer-node1.example.com         1/1       Running   0          28m
+If only the container image in a manifest has changed, update it.
```

Comment:
Let's put this note and "image only" option before the edit command option (so we start with simple option, then move to more detailed options).

So maybe a bit more description on the note as well

> If you have determined that the only change is the container image version (and not configuration, or flags, for example), then you can simply update the container image with a single command. This is more common in the case of patch version upgrades.

Then for the note about editing, maybe something like:

> If you are changing more of the manifest than just the image version, you can make the edits to the object directly